scripts/get-all-images.py

```diff
@@ -0,0 +1,223 @@
+import argparse
+import logging
+import shutil
+import subprocess
+import sys
+
+import git
+import yaml
+
+# logging
+LOG_FORMAT = "%(levelname)s \t| %(message)s"
+logging.basicConfig(format=LOG_FORMAT, level=logging.INFO)
+log = logging.getLogger(__name__)
+
+# consts
+EXCLUDE_CHARMS = ["kubeflow-roles"]
+
+GH_REPO_KEY = "_github_repo_name"
+GH_BRANCH_KEY = "_github_repo_branch"
+GH_DEPENDENCY_REPO_KEY = "_github_dependency_repo_name"
+GH_DEPENDENCY_BRANCH_KEY = "_github_dependency_repo_branch"
+GET_IMAGES_SH = "tools/get-images.sh"
+
+AIRGAP_TESTING_IMAGES = [
+    "charmedkubeflow/pipelines-runner:ckf-1.8",
+    "docker.io/kubeflowkatib/simple-pbt:v0.16.0",
+    "ghcr.io/knative/helloworld-go:latest",
+    "gcr.io/kubeflow-ci/tf-mnist-with-summaries:1.0",
+]
```

Comment:
these images are expected to change with every release, usually the tag will be a newer one.
It seems to me that there is no way around hardcoding them, wdyt?
If not, can you add a comment to make this clear?

Reply:
What if we provide these as input argument for script? Like that, we could only change it in workflow.
IMO, we should mention why those images are needed and how I can found out which to use for release. WDYT?

Reply:
I also think this could be an issue. I like @rgildein idea, perhaps they can be passed to the script as txt file? `get-all-images.py --airgap-testing-images=airgap-testing-images.txt` and we can place this file inside the `airgap` dir?
However the approach, please document when and how these should be updated.

Reply:
This will be more involved than stated above. The script for deploying in arigap will also need to know which images to use.

So either the script that deploys the whole airgap now also needs to be aware of this, or the new python script will include all the images by default.

Reply:
Pushed a commit for this ee54b85

The overview is:
1. The `scripts/get-all-images.py` script will have a new `--append-images` argument, path to the images file
2. The `tests/airgapped/airgap.sh` script will need a new `--testing-images-path` arg, which is used when gathering images
3. Updated the README files to mention of the above flow
4. Introduced a new `tests/airgapped/ckf-1.8-testing.txt` file containing all the images for airgap testing 1.8

Reply:
that's much better! I have a few comments that I'm leaving in the next review, so I'll resolve this for now and we can discuss there